Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1241,6 +1242,9 @@ private void OnEnable()
                 Active = true;
                 Active = false;
             }
+
+            //Subcribing to pointer events bubbling up from handlers
+            CoreServices.InputSystem?.RegisterPropagationHandler<IMixedRealityPointerHandler>(this);
```

Comment:
Does registering with no parameters register for bubble up? I noticed that you explicitly need to specify propagation phase in unregister but not in register.

Reply:
default parameter for register is PropagationPhase.BubbleUp - i didn't realize there's no default for unregister - we probably would want to add the same default or remove default entirely.

Reply:
Yeah... it makes sense. changing that.